docs/api/main-config-config.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'config.config'
+---
+
+Type: `string[] | ((config: string[], options: Options) => string[] | Promise<string[]>)`
+
+Deprecated: `true`
```

Comment:
While I don't think the guides or tutorials should be obligated to include deprecated features, I think the API references should.